repos/system_upgrade/el7toel8/models/hardwareinfo.py

```diff
@@ -0,0 +1,23 @@
+from leapp.models import Model, fields
+from leapp.topics import SystemFactsTopic
+
+
+class HardwareInfo(Model):
+    """
+    The model represents information about Hardware
+
+    The model currently doesn't represent all information about hardware could
+    provide on the machine. Just part of them, in case any other attributes
+    will be needed, the model can be extended.
+    """
+
+    topic = SystemFactsTopic
+
+    mem_total = fields.Nullable(fields.Integer())
+    """
+    Total memory in kib
+    """
+    kernel_version = fields.Nullable(fields.String())
+    """
+    Kernel version
+    """
```

Comment:
How the version of the kernel is supposed to represent HW information? As well - the oldest kernel provided for RHEL 7 has version 3.10. Why to test this in that case? Additionaly it's duplicit information. As well - description doesnt fit. Information about CPU is information about hardware and it is not part of this model, but different one.